IdentityCore/src/MSIDConstants.h

```diff
@@ -35,3 +35,4 @@ extern NSString * _Nonnull const MSIDTrustedAuthorityGermany;
 extern NSString * _Nonnull const MSIDTrustedAuthorityWorldWide;
 extern NSString * _Nonnull const MSIDTrustedAuthorityUSGovernment;
 extern NSString * _Nonnull const MSIDTrustedAuthorityCloudGovApi;
+extern NSString * _Nonnull const MSIDServerErrorClientMismatch;
```

Comment:
If this is a server error, then it should be at MSIDError.

Reply:
This is a server sub error so I just added it in MSIDConstants. The main error category is still interaction_required and we already have a category for it.